src/SGEMM/TropicalSGemm.cu

```diff
@@ -0,0 +1,622 @@
+#include <cstdint>
```

Comment:
If it is adapted from code from somewhere else, you'd better credit that repo on the top of this file.